modules/search/instant-search/components/search-results.scss

```
@@ -5,6 +5,16 @@
 	max-width: 1080px;
 	min-height: 400px;
 	text-align: left;
+
+	mark {
+		background: none;
+		font-weight: bold;
+		color: inherit;
```

```suggestion
		color: inherit;
		padding: 0;
```

It might be worth adding `border` and `margin` rules here, too.